README.md

```diff
@@ -1,13 +1,36 @@
-Increment version number in setup.cfg
+# PlaygroundRL Environments Package
 
-Build with 
-`python -m build` 
+This repository contains the game environments used in the backend
+of the PlaygroundRL website. These environments help to facilitate
+gameplay by providing relevant game boundaries, rules, and states.
+Each environment included is game-specific, defining relevant
+classes and functions needed to characterize gameplay.
+In particular, the repository contains PettingZoo environments,
+allowing for multi-agent reinforcement learning research opportunities. 
+Instructions to install this package are below. 
+
+## How to Install
```

Comment:
This is actually the instructions on how to build. How to install is much simpler: 

```
pip install playgroundrl-envs
```